### PR TITLE
NEXUS-24271: aligned Helm ITs with renewed IT infrastructure

### PR DESCRIPTION
**Pull request by igorudovika**
NEXUS-24271: aligned Helm ITs with renewed IT infrastructure

### CODE REVIEW

File: nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmHostedIT.java [hunk 1]
```diff
@@ -69,15 +70,24 @@ public void setUp() throws Exception {
...
-    Component component = findComponent(repository, MONGO_PKG_NAME);
-    assertThat(component.name(), is(equalTo(MONGO_PKG_NAME)));
-    assertThat(component.version(), is(equalTo(MONGO_PKG_VERSION_600)));
-
-    //Verify Asset is created.
-    Asset asset = findAsset(repository, MONGO_PATH_FULL_600_TARGZ);
-    assertThat(asset.name(), is(equalTo(MONGO_PATH_FULL_600_TARGZ)));
-    assertThat(asset.format(), is(equalTo(HELM_FORMAT_NAME)));
+    //Verify DB contains data about uploaded components and assets
+    Assert.assertTrue(componentAssetTestHelper.componentExists(repository, MONGO_PKG_NAME, MONGO_PKG_VERSION_600));
```
Comment (Darkvar) [inline nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmHostedIT.java:74]: Would you mind to use static import for `assertXXX` methods? Also, please remove unused imports.

File: nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmProxyIT.java [hunk 2]
```diff
@@ -65,61 +68,45 @@ public void tearDown() throws Exception {
...
-    Assert.assertThat(nullComponent, is(nullValue()));
+    Assert.assertFalse(componentAssetTestHelper.componentExists(repository, MONGO_PKG_NAME, MONGO_PKG_VERSION_600));
     client.fetch(MONGO_PKG_FILE_NAME_600_TGZ, CONTENT_TYPE_TGZ);
-
-    final Component component = findComponent(repository, MONGO_PKG_NAME);
-    Assert.assertThat(component.name(), is(equalTo(MONGO_PKG_NAME)));
-    Assert.assertThat(component.format(), is(equalTo(HELM_FORMAT_NAME)));
-    Assert.assertThat(component.group(), is(nullValue()));
-    Assert.assertThat(component.version(), is(equalTo(MONGO_PKG_VERSION_600)));
+    Assert.assertTrue(componentAssetTestHelper.componentExists(repository, MONGO_PKG_NAME, MONGO_PKG_VERSION_600));
```
Comment (Darkvar) [inline nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmProxyIT.java:83]: static import and remove unused imports please

File: nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/HelmHostedFacet.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.helm.internal.hosted;
```
Comment (Darkvar) [inline nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/HelmHostedFacet.java:13]: looks like package name is not corresponding to the real location.

File: nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmHostedIT.java (hunk 1 above)
Comment (igorudovika) [reply nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmHostedIT.java]: fixed 8402364654feb6ef462e10fada1794abf0a580c6

File: nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmProxyIT.java (hunk 2 above)
Comment (igorudovika) [reply nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmProxyIT.java]: fixed 8402364654feb6ef462e10fada1794abf0a580c6

File: nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/HelmHostedFacet.java (hunk 3 above)
Comment (igorudovika) [reply nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/HelmHostedFacet.java]: ups, sorry, from another PR.
Fixed 8402364654feb6ef462e10fada1794abf0a580c6

File: nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmHostedIT.java [hunk 4]
```diff
@@ -40,12 +37,15 @@
...
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
 import static org.sonatype.nexus.plugins.helm.HelmITConfig.configureHelmBase;
 import static org.sonatype.nexus.repository.http.HttpStatus.NOT_FOUND;
 import static org.sonatype.nexus.testsuite.testsupport.FormatClientSupport.status;
 
+//import org.junit.Assert;
```
Comment (chernikovmaksym) [inline nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmHostedIT.java:47]: Uncomment/remove?

Comment (igorudovika) [reply nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmHostedIT.java]: ups, thanks. Fixed 4c1b69de2bba76bb1b37614b8c4b6d138eda8e86

**Pull request merged**